vermouth/processors/do_links.py

```diff
@@ -292,5 +293,10 @@ def run_molecule(self, molecule):
                         interaction = _build_link_interaction_from(molecule, interaction, match)
                         molecule.add_or_replace_interaction(inter_type, *interaction, link.citations)
 
+                for loglevel, entries in link.log_entries.items():
+                    for entry, fmt_args in entries.items():
+                        fmt_args = fmt_args + [match]
+                        molecule.log_entries[loglevel][entry] += fmt_args
```

Comment:
I'm a bit concerned about performance, but if we have not too many debug statements in links for now I think it is fine.

Reply:
I did some premature optimization, and populate the log_entries attribute only if they'd result in actual log messages being emitted.